src/scss/constants.scss

```diff
@@ -214,7 +216,9 @@ $z-layer-embed-iframe: $z-layer-on-top-bg;
 $z-layer-external-message: $z-layer-on-top-bg;
 $z-layer-auditorium-iframe: $z-layer-on-top-bg;
 $z-layer-jukebox: $z-layer-on-top-bg;
+$z-layer-popover: $z-layer-on-top-bg;
 $z-layer-start-table: 0;
+$z-layer-modal: 400;
```

Comment:
These work best if they're relative, like saying "Y should be above X, regardless what the actual z-index of X " ( y = x + 400).

Reply:
now uses `footer + 10` value c90aa28